src/server/scripts/EasternKingdoms/ScarletEnclave/chapter2.cpp

```diff
@@ -1027,6 +1027,44 @@ class spell_chapter2_persuasive_strike : public SpellScript
     }
 };
 
+enum AcherusPortal
+{
+    SPELL_PORTAL_EFFECT_ACHERUS   = 53098,
+    QUEST_SCARLET_ARMIES_APPROACH = 12757
+};
+
+class spell_portal_effect_acherus : public SpellScript
+{
+    PrepareSpellScript(spell_portal_effect_acherus);
+
+    bool Validate(SpellInfo const* /*spellInfo*/) override
+    {
+        return ValidateSpellInfo({ SPELL_PORTAL_EFFECT_ACHERUS });
+    }
+
+    SpellCastResult CheckCast()
+    {
+        Unit* target = GetExplTargetUnit();
+        if (target && target->IsPlayer() && target->ToPlayer()->HasQuest(QUEST_SCARLET_ARMIES_APPROACH))
+            return SPELL_CAST_OK;
+
+        return SPELL_FAILED_DONT_REPORT;
+    }
+
+    void HandleScriptEffect(SpellEffIndex effIndex)
```

Comment:
This is throwing an error 

fatal error: unused parameter 'effIndex' [-Wunused-parameter]
    void HandleScriptEffect(SpellEffIndex effIndex)